Reporting/ad-extensions-reporter.js

```diff
@@ -0,0 +1,395 @@
+/**
+ *
+ * Ad Extensions Reporter
+ *
+ * Outputs to a spreadsheet all the ad extensions in an account
+ *
+ * Version: 1.0
+ * Google Ads Script maintained on brainlabsdigital.com
+ *
+ **/
+
+////////////////////////////////////////////////////////////////////////////////
+// Options
+
+var spreadsheetUrl = "https://docs.google.com/YOUR-SPREADSHEET-URL-HERE";
+// The URL of the Google Doc the results will be put into.
+
+var includeExtensionsWithZeroImpressions = true;
+// Keep as true if you also want to see the extensions that aren't being shown. This is the default,
+// and recommended if you are using this script to get an overview of all the extensions in your account.
+// In accounts with a very large number of extensions, this script is too slow. If that is the case
+// for your account, you can change this variable's value to false to speed it up. That will exclude
+// extensions with no impressions today.
```

Comment:
Just cut down a few words to make it shorter, good comment though!
```suggestion
// Keep as true if you also want to see the extensions that aren't being shown. This is the default,
// and recommended if you are using this script to get an overview of all the extensions in your account.
// (In accounts with a very large number of extensions, this script is too slow, so you can change this to 
// false to speed it up. That will exclude extensions with no impressions today.)
```

Reply:
Yes that's better!